nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ParseSyslog.java

```diff
@@ -57,13 +57,17 @@
 @SupportsBatching
 @InputRequirement(Requirement.INPUT_REQUIRED)
 @Tags({"logs", "syslog", "attributes", "system", "event", "message"})
-@CapabilityDescription("Parses the contents of a Syslog message and adds attributes to the FlowFile for each of the parts of the Syslog message")
+@CapabilityDescription("Attempts to parses the contents of a Syslog message in accordance to RFC5424 and RFC3164 " +
+        "formats and adds attributes to the FlowFile for each of the parts of the Syslog message." +
+        "Note: Be mindfull that RFC3164 is informational and a wide range of different implementations are present in" +
+        " the wild. If messages fail parsing, considering using RFC5424 or using a generic parsin processors such as " +
+        "ExtractGrok.")
 @WritesAttributes({@WritesAttribute(attribute = "syslog.priority", description = "The priority of the Syslog message."),
     @WritesAttribute(attribute = "syslog.severity", description = "The severity of the Syslog message derived from the priority."),
     @WritesAttribute(attribute = "syslog.facility", description = "The facility of the Syslog message derived from the priority."),
     @WritesAttribute(attribute = "syslog.version", description = "The optional version from the Syslog message."),
     @WritesAttribute(attribute = "syslog.timestamp", description = "The timestamp of the Syslog message."),
-    @WritesAttribute(attribute = "syslog.hostname", description = "The hostname of the Syslog message."),
+    @WritesAttribute(attribute = "syslog.hostname", description = "The hostname or IP address of the Syslog message."),
```

Comment:
Should we update this on ListenSyslog as well since the same parsing will be used there?